textattack/loggers/file_logger.py

```diff
@@ -43,6 +45,8 @@ def __setstate__(self, state):
     def log_attack_result(self, result):
         self.num_results += 1
         color_method = "ansi" if self.stdout else "file"
+        if self.nocolor:
+            color_method = None
```

Comment:
I think it's better to set `color_method` to be `"file"` instead of `None`. `None` leaves no marks as to which words are different, but `"file"` marks it using double brackets ("[[ <word> ]]"). 

Also the logic needs to be clearer. Nocolor option should be check if `self.stdout` is true. Ex:
```
if self.stdout:
    color_method = "file" if self.nocolor else "ansi"
else:
    color_method = "file"
```

Reply:
Could you also add the part the automatically enforces no color option when redirecting stdout? You can check via `sys.stdout.isatty()`. 

Also, for redirection case, could you also update this part https://github.com/QData/TextAttack/blob/994a56caf29c7e0162d8fa5ec5d58eab58d30daa/textattack/shared/utils/install.py#L95 so that there's no coloring around "textattack" prefix when we're redirecting?